server/services/authorization/canAccessRoom.ts

```diff
@@ -17,6 +18,23 @@ async function canAccessPublicRoom(user: Partial<IUser>): Promise<boolean> {
 	return Authorization.hasPermission(user._id, 'view-c-room');
 }
 
+const roomViewPermission = async (roomType: RoomType, uid: string): Promise<boolean> => {
```

Comment:
```suggestion
const hasRoomViewPermission = async (roomType: RoomType, uid: string): Promise<boolean> => {
```
Maybe we could change this function name to improve legibility

Reply:
Updated